language/diem-framework/modules/XDX.move

```diff
@@ -98,13 +98,14 @@ module XDX {
         Diem::is_currency<CoinType>() &&
```

Comment:
I think the specs would be simpler if this function just aborted when CoinType is not a currency. Can we do that @tzakian?

Reply:
Perhaps not for this release, we really appear to have a tight workload for this one. Open an issue?

Reply:
I I don't think we should change this to an assertion at least for this upcoming release since it feels a bit odd to me to have a predicate about a property abort because one of those properties isn't satisfied. But, happy to discuss some more and update after this release.